doc/ven_impl/ven_template/README.md

```diff
@@ -0,0 +1,40 @@
+# CNTT Vendor Implementation - VI
+
+## Vendor Name
+Vendor XX
+
+## Target Reference Architecture :
+|   Type          | Release  | Notes
+|--------------------------------|--------------------------------|-------------------------------|
+| OpenStack (RA-1) |  v3.0   |   |
```

Comment:
Are the releases version #s or the names we have been assigning. Personally I prefer Version numbers.

Reply:
Yes they are version numbers